Uno.Gallery/Uno.Gallery.UWP/Views/Styles/XamlDisplay.xaml

```diff
@@ -170,115 +122,69 @@
 
 	<Style x:Key="DefaultXamlPresenterStyle"
 		   TargetType="smtx:XamlPresenter">
-        <Setter Property="Formatter"
-                Value="{StaticResource NiceFormatter}" />
-        <Setter Property="Template">
+		<Setter Property="Formatter"
+				Value="{StaticResource NiceFormatter}" />
+		<Setter Property="Template">
 			<Setter.Value>
 				<ControlTemplate TargetType="smtx:XamlPresenter">
 					<Grid>
 						<Grid.RowDefinitions>
 							<RowDefinition Height="Auto" />
-							<RowDefinition Height="*" />
+							<RowDefinition Height="Auto" />
+							<RowDefinition Height="Auto" />
 						</Grid.RowDefinitions>
 
 						<ContentControl Content="{Binding Tag, RelativeSource={RelativeSource Mode=TemplatedParent}}"
-                                        VerticalContentAlignment="Center"
-										android:Visibility="Collapsed"
-										ios:Visibility="Collapsed" />
+										VerticalContentAlignment="Center" />
+
+						<Grid Grid.Row="1"
+							  Padding="16,0,6,0"
```

Comment:
Padding="16,0,6,0" use here instead of Padding="16,0" to compensate the RevealButton padding
![image](https://github.com/unoplatform/Uno.Gallery/assets/16295702/4485c5b3-98d7-4f80-983c-c01dd2286e51)

Reply:
could use 16,0,0,0 here,
so it wont look as weird when onclick effect is active:
![image](https://github.com/unoplatform/Uno.Gallery/assets/2359550/a22b7b70-fc30-4728-b7fc-a4e27411064d)

Reply:
Adjusted, thanks ;)